src/index.html

```diff
@@ -17,6 +20,12 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch stopwatch--speed-up">
+      <div class="hand">
```

Comment:
The 'hand' element is missing the parent block class 'stopwatch__hand' if it's intended to be a BEM element of 'stopwatch'.